options.go

```diff
@@ -529,6 +534,50 @@ func parseSignatureKey(o *Options, msgs []string) []string {
 	return msgs
 }
 
+func parseClaimAuthorizations(o *Options, msgs []string) []string {
+
+	v := &JMESValidator{}
+
+	if o.ClaimAuthorization != "" {
+		if _, err := v.AddRule(o.ClaimAuthorization); err != nil {
+			msgs = append(msgs, fmt.Sprintf("%v", err))
+		}
+	}
```

Comment:
Again, this looks like only one value can come from the flag 🤔 

I'm actually wondering if users should only be able to provide them by file, we have a lot of flags already, might be neater to just set this via a separate file

Reply:
Forcing into a file is very inconvenient when configuring via cloud services in some cases. For example when just using an "off-the-shelf" Docker image. Agreed on too many command line options in general, though...